src/plugins/intel_cpu/src/emitters/snippets/cpu_runtime_configurator.hpp

```diff
@@ -44,13 +44,28 @@ class CPURuntimeConfigurator : public ov::snippets::RuntimeConfigurator {
      * @param linear_ir LinearIR
      */
     void init_tensor_rank(const ov::snippets::lowered::LinearIRCPtr& linear_ir) const override;
+    void initialization(const ov::snippets::lowered::LinearIRCPtr& linear_ir) override;
     /**
      * @brief Calculate Loop parameters of Loop emitters and update these values in CPURuntimeConfig
      * @param linear_ir LinearIR
      */
     void update_loop_args(const ov::snippets::lowered::LinearIRCPtr& linear_ir) const;
 
     static const size_t rank6D;
+
+    class BrgemmCopyBLoopPortsAdjuster {
+    public:
+        BrgemmCopyBLoopPortsAdjuster() = default;
+        BrgemmCopyBLoopPortsAdjuster(const ov::snippets::lowered::LinearIRCPtr& linear_ir,
+                                     ov::intel_cpu::CPURuntimeConfigurator *configurator);
+
+        void optimize();
+
+    private:
+        using UniLoopInfoPtr = std::shared_ptr<snippets::lowered::UnifiedLoopInfo>;
```

Comment:
We can, but it would be much less readable because we'll have to specify the `snippets::lowered` namespace every time.
I changed it, please take a look